exporters/otlp/src/otlp_file_client.cc

```diff
@@ -20,6 +20,7 @@
 #include "opentelemetry/exporters/otlp/protobuf_include_suffix.h" // IWYU pragma: keep
 // clang-format on
 
+#include "opentelemetry/common/macros.h"
```

Comment:
macros.h is not needed.

By convention, including version.h also provides macros.h, because it contains the magic line:

```
#include "opentelemetry/common/macros.h"  // IWYU pragma: export
```

This was done because every file depends on version.h anyway, so code can reliably make use of various defines like OPENTELEMETRY_HAVE_EXCEPTIONS without having to remember to include macros.h.

Reply:
Reported by include-what-you-use as:

```
/home/runner/work/opentelemetry-cpp/opentelemetry-cpp/exporters/otlp/src/otlp_file_client.cc should remove these lines:
- #include "opentelemetry/common/macros.h"  // lines 23-23
```

because of the export pragma in version.h.

Reply:
Thanks, it's removed now.